src/components/passwordreset/components/input.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Input from './input';
+
+describe('test input element component', () => {
+  it('should test the input component', () => {
+    shallow(<Input name="email" type="email" handleChange={() => ''} />);
```

Comment:
@Anguandia I suggest we expect something to equal some test matcher.

Reply:
@minega25 , changed the individual component tests to check specific component element properties, thanks for the observation